app/assets/javascripts/routes/products_handlers.js

```diff
@@ -99,6 +100,11 @@ let ProductsHandlers = {
       type: ActionTypes.PRODUCT_RECEIVE,
       product: product
     });
+
+    Dispatcher.dispatch({
```

Comment:
Expected ',' and instead saw '.'.<br>Expected ',' and instead saw '('.<br>Expected an identifier and instead saw '{'.